atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -3,8 +3,8 @@ scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
 default_scan: redhat-release
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
```

Comment:
I'd suggest to use a name like `yum-update` or `rpm-update` since that's what this scan does. Using the name `pipeline-scanner` wouldn't indicate to the user as to what it will do.